android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/Settings.kt

```diff
@@ -241,6 +241,11 @@ abstract class Settings {
      * Setting the HTTPS-Only mode for upgrading connections to HTTPS.
      */
     open var httpsOnlyMode: Engine.HttpsOnlyMode by UnsupportedSetting()
+
+    /**
+     * Setting o control whether Global Privacy Control isenabled.
```

Comment:
nit: Setting **to** control whether Global Privacy Control **is** enabled.

Reply:
🦅 👁️